framework/src/microservice.js

```diff
@@ -122,29 +122,43 @@ const Microservice = (config = {}) => {
 				`${util.inspect(job)}`,
 				`${util.inspect(validDefinition)}`,
 			].join('\n'));
-			return;
+			return false;
```

Comment:
Returning of `true` or `false` should be consistent across `addMethod` and `addEvent` as well.

Reply:
Done!